util/test/checkCopyrights.bash

```diff
@@ -18,6 +18,15 @@
 #  Make*
 #
 
+todate=`date "+%m%d"`
+#This hardcoding is intentional. Date accepts a full date like this, but %m%d cuts off the year so it only compares 0107 to the current date
+cond=`date -d 2022-01-07 "+%m%d"`
```

Comment:
Doing some post-merging sanity checking after the merge #18920 , I'm wondering:  Was there no way to write this such that the grace period was agnostic to the year?  E.g., as it stands, I think we'll have to update this script each year to re-enable to grace period, won't we?

Reply:
There was a comment on that already, see [this convo](https://github.com/chapel-lang/chapel/pull/18896#discussion_r774205701)

Reply:
Is the conclusion of that conversation simply there's known work to do here still?  Is there an issue describing that work somewhere?  I'm imagining that we'd just query the current year into a variable and jam that into where 2022 is hardcoded today?  (i.e., doesn't seem challenging to fix unless I'm missing something—the kind of case where I'd probably just fix the issue rather than filing it).

Reply:
The conclusion is that the year doesn't actually impact anything, since the `+%m%d` only grabs the day and year from it

Reply:
Ohhh, OK, I didn't understand that.  And now I see the comment just prior to the line I commented on says exactly this (my eyes never see comments).  Thanks for clarifying!